DSharpPlus.Rest/DiscordRestClient.cs

```diff
@@ -472,7 +472,7 @@ public Task<DiscordMessage> CreateMessageAsync(ulong channel_id, string content)
         /// <param name="embed">Embed to attach</param>
         /// <returns></returns>
         public Task<DiscordMessage> CreateMessageAsync(ulong channel_id, DiscordEmbed embed)
-            => this.ApiClient.CreateMessageAsync(channel_id, null, new[] {embed}, replyMessageId: null, mentionReply: false, failOnInvalidReply: false);
+            => this.ApiClient.CreateMessageAsync(channel_id, null, new[] { embed }, replyMessageId: null, mentionReply: false, failOnInvalidReply: false);
```

Comment:
No null check here?

Reply:
Here you're not supposed to be supplying null anyway, so it didn't make sense to add, and in that case a nullref would give more information than the `BadRequestException` discord would send